state_num: 3
control_num: 2

#f_imu_to_cam: [ 0.99819265 -0.05060387 -0.03241437 -0.00095301, 0.0336592   0.02394397  0.99914651  0.00032153, -0.04978455 -0.99843174  0.02560398 -0.00111187, 0., 0., 0., 1.]
